com.unity.render-pipelines.universal/ShaderLibrary/Core.hlsl

```diff
@@ -39,20 +39,20 @@
 
 
 #if UNITY_REVERSED_Z
-    #if SHADER_API_OPENGL || SHADER_API_GLES || SHADER_API_GLES3
-        //GL with reversed z => z clip range is [near, -far] -> should remap in theory but dont do it in practice to save some perf (range is close enough)
-        #define UNITY_Z_0_FAR_FROM_CLIPSPACE(coord) max(-(coord), 0)
+    #if SHADER_API_GLCORE || SHADER_API_GLES || SHADER_API_GLES3
```

Comment:
This path is dead code isn't it? OpenGL doesn't support reversed Z.

Reply:
Currently as far as I know. But was there so I left it there, as REVERSE_Z doesn't exactly mean OpenGL (even if that is the case in practice.)
Do you want remove it?